src/components/messenger/list/conversation-item.tsx

```diff
@@ -153,7 +153,7 @@ export class ConversationItem extends React.Component<Properties> {
             </div>
             <div className={c('content')}>
               <div className={c('message')} is-unread={isUnread}>
-                <ContentHighlighter message={this.message} />
+                <ContentHighlighter message={this.message} variant='negative' />
```

Comment:
Not sure `negative` holds any meaning to me...but I also can't come up with a suggestion. haha